routes/markets.js

```diff
@@ -1,8 +1,36 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const firebase = require('firebase');
+const config = require('./config.js');
+const router = express.Router();
+
+if (!firebase.apps.length) {
+	firebase.initializeApp(config.config);
+}
+// Creates connection to database.
+const db = firebase.database();
+// Links to head of database.
+const ref = db.ref("live_weller");
+// Links to markets list.
+const marketsRef = ref.child("markets");
 
 router.get('/', function(req, res, next) {
-	res.render('markets');
+	let markets = [];
+
+	marketsRef.once('value', function(snapshot) {
+		snapshot.forEach(function(childSnapshot) {
+			let childData = childSnapshot.val();
+
+			let name = childData.marketInfo.marketName;
+			let address = childData.marketInfo.address;
+			let size = childData.marketInfo.storeType;
+			let zip = childData.marketInfo.zip;
+			let level = childData.marketInfo.marketLevel;
+
+			markets.push({name: name, address: address, size: size, zip: zip, level: level});
```

Comment:
You can fold the variable declarations and JSON object into one: 
```javascript
markets.push({
    name: childData.marketInfo.marketName
});
```

Also, if the key name and value variable are the same, e.g. `{ level: level }`, you don’t have to specify the value (`{ level: level }` is equivalent to `{ level }`).